QNormalizModule.cpp

```diff
@@ -207,7 +207,8 @@ bool PyNumberToNmz( PyObject * in, mpq_class& out ){
         return true;
     }
     PyObject * in_as_string = PyObject_Str( in );
-    const char* in_as_c_string = PyUnicodeToString( in_as_string ).c_str();
+    string s = PyUnicodeToString( in_as_string );
+    const char* in_as_c_string = s.c_str();
```

Comment:
Good catch. `s` might be prematurely deleted. I looked at the C++-Reference, and indeed, the return value of `c_str` points to the internal memory of the string object.